js/calc.js

```diff
@@ -2,44 +2,72 @@
 document.addEventListener("DOMContentLoaded", function(event){
 
 var screen;
-var	output
-var operation;
+var	result;
+//The function button
+var operator;
+//The number currently being entered
+var current;
+//to store the previous number you entered
+var memory;
 
 var screen = document.getElementById("screen");
 
-//now we want to click on number in a calculator, it's targeting
+//now we want to click on number in the calculator
 var buttonNum = document.getElementsByClassName('buttonNum');
-    for(var i = 0; i < buttonNum.length; i++ ) {
-        buttonNum[i].addEventListener("click",function(event) {    
+for(var i = 0; i < buttonNum.length; i++ ) {
+	buttonNum[i].addEventListener("click",function(event) {    
             //console.log(event.target.innerHTML);
             screen.innerHTML = event.target.innerHTML;
-       });
-    }
+        });
+}
 
 var operator = document.getElementsByClassName('operator');
-    for(var i = 0; i < operator.length; i++ ) {
-        operator[i].addEventListener("click",function(event) {
-            screen.innerHTML = event.target.innerHTML;
-       });
-    }
+for(var i = 0; i < operator.length; i++ ) {
+	operator[i].addEventListener("click",function(event) {
+		screen.innerHTML = event.target.innerHTML;
+	});
+}
 
 var equals = document.getElementsByClassName('equals');
-    for(var i = 0; i < equals.length; i++ ) {
-        equals[i].addEventListener("click",function(event) {
-            screen.innerHTML = event.target.innerHTML;
-       });
-    }
+for(var i = 0; i < equals.length; i++ ) {
+	equals[i].addEventListener("click",function(event) {
+		screen.innerHTML = event.target.innerHTML;
+	});
+}
 
 var buttonClear = document.getElementsByClassName('buttonClear');
-    for(var i = 0; i < buttonClear.length; i++ ) {
-        buttonClear[i].addEventListener("click",function(event) {
-            screen.innerHTML = event.target.innerHTML;
-       });
-    }
-
+for(var i = 0; i < buttonClear.length; i++ ) {
+	buttonClear[i].addEventListener("click",function(event) {
+		screen.innerHTML = event.target.innerHTML;
+	});
+}
 
 
+// Operation functions
+function Calculator() {
+	var operator = function(result, input) {
+		return result + input;
```

Comment:
This function doesn't make any sense. When it hits return here it ends, it won't run any code below it.